src/3box.js

```diff
@@ -11,8 +11,27 @@ const OrbitdbKeyAdapter = require('./orbitdbKeyAdapter')
 const utils = require('./utils')
 
 const ADDRESS_SERVER_URL = 'https://beta.3box.io/address-server'
-const PINNING_SERVER = '/ip4/18.236.233.237/tcp/4003/ws/ipfs/QmbgpyTLCYBy84E1HGwei6niQHiLQmRpfN6SQtfJiyNMUd'
+const PINNING_SERVER = '/dnsaddr/ipfs.3box.io/tcp/443/wss/ipfs/QmbgpyTLCYBy84E1HGwei6niQHiLQmRpfN6SQtfJiyNMU'
 const IPFS_OPTIONS = {
+  config: {
+    Addresses: {
+      Swarm: [
+        '/dns4/wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star',
+        '/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star'
```

Comment:
What are these? Looks like some webrtc enablers? Do you know what they do?

Reply:
Only the first one seems to be a webrtc enabler. I think it's safe to remove them after testing they work. But I don't find harmful them to be there.

Reply:
let's keep for now, we can test in #82